src/main/java/net/twasi/obsremotejava/message/event/Event.java

```diff
@@ -112,6 +118,10 @@ protected Event(Type eventType, Category category) {
         MediaInputs(1 << 8),
         // Receive all event categories (default subscription setting)
         All(General.value | Config.value | Scenes.value | Inputs.value | Transitions.value | Filters.value | Outputs.value | SceneItems.value | MediaInputs.value),
+        // InputActiveStateChanged event (high-volume)
+        InputActiveStateChanged(1 << 10),
```

Comment:
1. Where did you get those values?
2. Shouldn't wee add it to the All?

Reply:
1. [obs-websocket EventSubscriptions Enum](https://github.com/Palakis/obs-websocket/blob/master/docs/generated/protocol.md#eventsubscriptions-enum)
2. Yes forgot to do that. will do it

Reply:
1. Ah yeah, nice, they edited it, it was not there when I started implementing
2. It is a question, I'm not sure we should

Reply:
2. Because if you use All you expect to get all events including the input events?

Reply:
Asked it in the obs-websocket discord and they say it should not be in there because those are high volume events so those need to be enabled explicitly.